tests/contrib/flask/test_request.py

```diff
@@ -163,9 +164,10 @@ def index():
         # Request tags
         assert spans[0].get_tag(http.QUERY_STRING) == "foo=bar&baz=biz"
 
+    @pytest.mark.skipif(flask_version >= (2, 0, 0), reason="Decoding error thrown in url_split")
```

Comment:
@gnufede  originally had this set to skip for `2,2,0` but I was still getting a decoding error with `2,0,0`.